presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -163,4 +166,33 @@ public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsPr
         this.awsCredentialsProvider = Optional.ofNullable(awsCredentialsProvider);
         return this;
     }
+
+    @Min(1)
+    @Max(10)
+    public int getTotalSegments()
+    {
+        return totalSegments;
+    }
+
+    @Config("hive.metastore.glue.total-segments")
```

Comment:
i don't think "total" should be here.

@electrum, `hive.metastore.glue.partitions-segments` ?

Reply:
Please let me know if this needs a change.